### PR TITLE
Issue #12082: Github action Publish release notes Twitter 

### LINKED ISSUE #12082: Github action "Publish release notes outside"
https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams-manual-trigger.md

we need to make action to do `run Github action "Publish release notes outside '10.3.1'"`  (action takes human provided version as input, just to make sure it can not be executed unintentionally)
if user provider input is not matching pom - fail build

so we simply make/reuse  scripts that does twit (need to know howto share secrets)
and upadate release notes to Github (existing script)

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes #12082 

To use Twitter variables as secrets: https://docs.github.com/en/actions/security-guides/encrypted-secrets

### CODE REVIEW

File: .github/workflows/copy_of_publish_releasenotes_outside.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#############################################################################
```
Comment (romani) [inline .github/workflows/copy_of_publish_releasenotes_outside.yml:1]: Rename file

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/copy_of_publish_releasenotes_outside.yml (hunk 1 above)
Comment (Rahulkhinchi03) [reply .github/workflows/copy_of_publish_releasenotes_outside.yml]: Done

**Review by romani — APPROVED** [review]
Ok to merge

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**